Presentation.Web/app/services/ItSystemUsage/it-system-usage-service.ts

```diff
@@ -5,30 +5,63 @@
         addDataLevel(systemUsageId: number, dataLevel: number);
         removeDataLevel(systemUsageId: number, dataLevel: number);
         patchSystemUsage(systemUsageId: number, orgId: number, payload: any);
+        patchPersonalData(systemUsageId: number, personalDataValue: Models.ViewModel.ItSystemUsage.PersonalDataOption): ng.IPromise<void>;
```

Comment:
rename to add -- since the other one is "remove"--- also param should not be a view model

Reply:
mayeb it's not a view model but actually the api enum?... then it is just the namespace which is wrong